model/model.js

```diff
@@ -48,11 +48,23 @@ ORDER BY created_at DESC`;
     return rows;
   });
 };
-// just adding comment to allow me to pull request
+createReviewComment = (review_id, username, body) => {
+  if (!username || !body || body.length === 0 || username.length === 0) {
```

Comment:
Although this is working these things really need to be handled by making a query to the DB with the given data and handling the PSQL errors that may come back. 👍

Reply:
I have removed this